### ISSUE TITLE
[a11y]: In AILabel component The 'id' "id-:r4h2:" specified for the ARIA property 'aria-controls' value is not valid

### ISSUE THREAD

**Issue by vasifrmn** [carbon-design-system/carbon#19691]
### Package

@carbon/react

### Browser

Chrome

### Operating System

MacOS

### Package version

1.81.0

### React version

18.2.0

### Automated testing tool and ruleset

IBM equal access accessibility checker

### Description

im geting The 'id' "id-:r4h2:" specified for the ARIA property 'aria-controls' value is not valid.

`<h5>
  {t.label}
<AILabel />
</h5>`

### Reproduction/example

https://stackblitz.com/edit/github-rqnkrx7z?file=src%2FApp.jsx&preset=node=

### Steps to reproduce

![Image](https://github.com/user-attachments/assets/dde9b6f6-5c06-417b-9b95-293abc506d47)

### Suggested Severity

None

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/carbon-design-system/carbon/blob/f555616971a03fd454c0f4daea184adf41fff05b/.github/CODE_OF_CONDUCT.md)
- [x] I checked the [current issues](https://github.com/carbon-design-system/carbon/issues) for duplicate problems

**Comment by ariellalgilmore** [comment]
Hi, looks like you are just missing the `AILabelContent`. The content is needed for the AILabel popover.

```
        <AILabel>
          <AILabelContent>Hello</AILabelContent>
        </AILabel>
```

 If you do not need a popover then I would recommend using the svg

```
import { AiLabel } from '@carbon/icons-react';

return <AiLabel />;

```
